hack/lib/util.sh

```diff
@@ -469,9 +469,11 @@ kube::util::ensure_no_staging_repos_in_gopath() {
 }
 
 # Installs the specified go package at a particular commit.
+# Optionally specify an additional dependency constraint as "pkg@revision".
 kube::util::go_install_from_commit() {
   local -r pkg=$1
   local -r commit=$2
+  local -r extra_constraint=${3:-}
```

Comment:
accept an array of these?

Reply:
eh, easy enhancement if we really need it, but I'd rather not encourage this. :)